configtx/application.go

```diff
@@ -467,6 +468,26 @@ func newApplicationGroup(application Application) (*cb.ConfigGroup, error) {
 	return applicationGroup, nil
 }
 
+// newApplicationGroupWithFullOrganizations returns the application component
+// of the channel configuration with the entire configuration for application
+// organizations.
+// By default, it sets the mod_policy of all elements to "Admins".
+func newApplicationGroupWithFullOrganizations(application Application) (*cb.ConfigGroup, error) {
```

Comment:
Maybe `newApplicationGroupTemplate` for the skeletal application group.

Reply:
Yeah, I didn't love the naming but I couldn't think of anything else really. Definitely agree it's better to rename this so that newApplicationGroup has everything and newApplicationGroupTemplate doesn't.

Reply:
Done